docs/src/pages/docs/functions/combinators.md

```diff
@@ -20,6 +20,32 @@ give it a value and it will give you back a function ready to take a function.
 Once that function is provided, it will return the result of applying your value
 to that function.
 
+```javascript
+import applyTo from 'crocks/combinators/applyTo'
+
+import map from 'crocks/pointfree/map'
+import compose from 'crocks/helpers/compose'
```

Comment:
lets put `compose` before `map`.

Reply:
So this will be in a document shortly but the import list is all alphabetical with three groups, subject, monoids/crocks, functions

```
import applyTo from 'crocks/combinators/applyTo'

import First from 'crocks/First'
import Pair from 'crocks/Pair'

import compose from 'crocks/helpers/compose'
import flip from 'crocks/combinators/flip'
import isArray from 'crocks/predicates/isArray'
import isNumber from 'crocks/predicates/isNumber'
import isString from 'crocks/predicates/isString'
import map from 'crocks/pointfree/map'
import merge from 'crocks/pointfree/merge'
import mreduceMap from 'crocks/helpers/mreduceMap'
import safeLift from 'crocks/Maybe/safeLift'
```